functions/__fzf_grep_current_dir.fish

```diff
@@ -0,0 +1,36 @@
+function __fzf_grep_current_dir --description "Search current directory content for $token. Replace the current token with the selected file paths."
+    # Make sure that fzf uses fish to execute __fzf_preview_file.
+    # See similar comment in __fzf_search_shell_variables.fish.
+    set --local --export SHELL (command --search fish)
+
+    set token (commandline --current-token | string unescape)
+
+    set fzf_arguments --multi --ansi --exact --phony --reverse --query=$token \
```

Comment:
What's the point of both `--exact --phony`? Also `--reverse` is already in the default args.

Reply:
I believe `--exact --phony` is needed to let ripgrep do the filtering. Here all filtering is provided by ripgrep, fzf is only a "GUI" on top.